cognite/powerops/custom_modules/power_model_v1/data_models/containers/SHOPResult.container.yaml

```diff
@@ -0,0 +1,100 @@
+space: '{{powerops_models}}'
+externalId: SHOPResult
+name: SHOPResult
+usedFor: node
+properties:
+  timeseries:
```

Comment:
Should we (if it is possible without too much overhead) add some "wrapping" around these time series? If we just have time series objects, we need to use time series metadata and/or name patterns to derive what each time series is for - would be nice to have an extended time series class with properties `object_type`, `object_name` and `attribute_name` (e. g. "plant", "Holen" and "production", respectively)